crates/ra_proc_macro/src/rpc.rs

```diff
@@ -34,15 +34,15 @@ pub struct ListMacrosResult {
 pub struct ExpansionTask {
     /// Argument of macro call.
     ///
-    /// In custom derive that would be a struct or enum; in attribute-like macro - underlying
+    /// In custom derive this will be a struct or enum; in attribute-like macro - underlying
     /// item; in function-like macro - the macro body.
     #[serde(with = "SubtreeDef")]
     pub macro_body: Subtree,
 
-    /// Names of macros to expand.
+    /// Names of macros to expand. // TODO: are they comma-separated?
```

Comment:
@Veetaha Would you mind to change the comment ?

Reply:
Sure https://github.com/rust-analyzer/rust-analyzer/pull/4061/commits/d8ca817456ed60b0163d3f81c58f6db9e6372e5f